client/app/bundles/course/statistics/pages/StatisticsIndex/course/StudentPerformanceTable.jsx

```diff
@@ -0,0 +1,520 @@
+      label: intl.formatMessage(translations.groupManagers),
+      options: {
+        filter: true,
+        filterType: 'multiselect',
+        filterOptions: {
+          names: [
+            ...new Set(
+              students.flatMap((s) => s.groupManagers.map((m) => m.name)),
+            ),
+          ],
+          logic: (managers, filters) => {
+            if (filters) {
+              const filterSet = new Set(filters);
+              return !managers
+                .map((m) => m.name)
+                .some((name) => filterSet.has(name));
+            }
+            return false;
+          },
+          fullWidth: true,
+        },
+        customFilterListOptions: {
+          render: (name) => `Tutor: ${name}`,
+        },
+        sort: false,
+        customBodyRenderLite: (dataIndex) => {
+          const groupManagers = displayedStudents[dataIndex].groupManagers;
+          if (!groupManagers) {
+            return '';
+          }
+          return (
+            <>
+              {groupManagers.map((m, index) => (
+                <span key={m.id}>
+                  <Link href={m.nameLink} opensInNewTab>
+                    {m.name}
+                  </Link>
+                  {index < groupManagers.length - 1 && ', '}
+                </span>
+              ))}
+            </>
+          );
+        },
+      },
+    });
+  }
+
+  if (isCourseGamified) {
+    columns.push({
+      name: 'level',
+      label: intl.formatMessage(translations.level),
+      options: {
+        filter: true,
+        sort: true,
+        sortDescFirst: true,
+        alignCenter: true,
+        filterType: 'multiselect',
+        filterOptions: {
+          fullWidth: true,
+        },
+        customFilterListOptions: {
+          render: (name) => `Level: ${name}`,
+        },
+      },
+    });
+    columns.push({
+      name: 'experiencePoints',
+      label: intl.formatMessage(translations.experiencePoints),
+      options: {
+        filter: false,
+        sort: true,
+        sortDescFirst: true,
+        alignCenter: true,
+        customBodyRenderLite: (dataIndex) => {
+          const student = displayedStudents[dataIndex];
+          return (
+            <Link
+              key={student.id}
+              href={student.experiencePointsLink}
+              opensInNewTab
+            >
+              {student.experiencePoints}
+            </Link>
+          );
+        },
+      },
+    });
+    columns.push({
+      name: 'achievementCount',
+      label: intl.formatMessage(translations.achievementCount),
+      options: {
+        filter: false,
+        sort: true,
+        sortDescFirst: true,
+        alignCenter: true,
+      },
+    });
+  }
+
+  columns.push({
+    name: 'numSubmissions',
+    label: intl.formatMessage(translations.numSubmissions),
+    options: {
+      filter: false,
+      sort: true,
+      sortDescFirst: true,
+      alignCenter: true,
+    },
+  });
+  columns.push({
+    name: 'correctness',
```

Comment:
Could you add more information about what it means by correctnes? Similar to the tooltip icon for the learning rate header.